modules/openy_features/openy_demo_content/modules/openy_demo_nclass/config/install/migrate_plus.migration.node_class_02.yml

```diff
@@ -1216,6 +1216,102 @@ source:
           sidebar_id: academic_success
+    -
+      id: class_cardio
+      title: 'Cardio'
+      activity: activity_cardio
+      body: |
+        <p>Cardio class.</p>
+    -
+      id: class_strength
+      title: 'Strength'
+      activity: activity_strength
+      body: |
+        <p>Strength class.</p>
+    -
+      id: class_forever_well
+      title: 'Forever Well:55+'
+      activity: activity_forever_well
+      body: |
+        <p>Forever Well class 55+.</p>
+    -
+      id: class_climbing_1
+      title: 'Climbing - Adult Rock Climbing - September'
+      activity:
+      body: |
+        <p>Climbing - Adult Rock Climbing - September</p>
+    -
+      id: class_climbing_2
+      title: 'Climbing - Adult Rock Climbing - October'
+      activity: activity_climbing_adult
+      body: |
+        <p>Climbing - Adult Rock Climbing - September</p>
+    -
+      id: class_climbing_3
+      title: 'Climbing - Parent/Child Climbing - September'
+      activity: activity_climbing_adult
+      body: |
+        <p>Climbing - Parent/Child Climbing - September </p>
+    -
+      id: class_climbing_4
+      title: 'Climbing - Parent/Child Climbing - September'
+      activity: activity_climbing_adult
+      body: |
+        <p>Climbing - Parent/Child Climbing - September</p>
+    -
+      id: class_cooking_1
+      title: 'Youth Cooking - Future Chefs'
+      activity: activity_cooking_classes
+      body: |
+        <p>Youth Cooking - Future Chefs</p>
+    -
+      id: class_aoa_1
+      title: 'AOA National Senior Health and Fitness Day'
+      activity: activity_health_wellness_workshops
+      body: |
+        <p>AOA National Senior Health and Fitness Day</p>
+    -
+      id: class_aoa_2
+      title: 'AOA Brain Agility - Class Fee, Spring 2017'
+      activity: activity_health_wellness_workshops
+      body: |
+        <p>AOA Brain Agility - Class Fee, Spring 2017</p>
+    -
+      id: class_aoa_3
+      title: 'AOA Brain Agility - Materials Fee, Spring 2017'
+      activity: activity_health_wellness_workshops
+      body: |
+        <p>AOA Brain Agility - Materials Fee, Spring 2017</p>
+    -
+      id: class_hi_1
+      title: 'LIVESTRONG at the YMCA'
+      activity: activity_health_improvement
+      body: |
+        <p>LIVESTRONG at the YMCA</p>
+    -
+      id: class_hi_2
+      title: 'Nutrition Consultation - Initial'
+      activity: activity_health_improvement
+      body: |
+        <p>Nutrition Consultation - Initial</p>
+    -
+      id: class_hi_3
+      title: 'ACT! Actively Changing Together'
+      activity: activity_health_improvement
+      body: |
+        <p>ACT! Actively Changing Together</p>
+    -
+      id: class_weight_1
+      title: 'Lose to Win for Men - Weight Loss Program'
+      activity: activity_weight_loss
+      body: |
+        <p>Lose to Win for Men - Weight Loss Program</p>
+    -
+      id: class_weight_2
+      title: 'Lose to Win - Weight Loss Program'
+      activity: activity_weight_loss
```

Comment:
![image](https://user-images.githubusercontent.com/563412/44039328-8c1e4696-9f21-11e8-98f2-a481bfec040b.png)
@danylevskyi we have to ensure we are not creating duplicates in content

Reply:
![image](https://user-images.githubusercontent.com/563412/44039410-b20a5ab6-9f21-11e8-93bd-e43015437eea.png)